tests/opentracing_instrumentation/test_postgres.py

```diff
@@ -22,17 +22,11 @@
 import psycopg2 as psycopg2_client
 import pytest
 from psycopg2 import extensions as pg_extensions, sql
-from sqlalchemy import (
-    Column,
-    Integer,
-    MetaData,
-    String,
-    Table,
-    create_engine,
-)
-from sqlalchemy.orm import mapper, sessionmaker
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
 
 from opentracing_instrumentation.client_hooks import psycopg2
+from .common import metadata, User
```

Comment:
"common" is not a good name. Can it be scoped better? Is it "sql_common" or "dbi_common"?

Reply:
I've renamed it to `sql_common`.
Thanks!